.github/workflows/osrm-backend.yml

```diff
@@ -34,12 +36,38 @@ jobs:
         node-version: 16
     - run: node --version
     - run: npm --version
+    - name: Prepare environment
+      shell: bash
+      run: |
+        PACKAGE_JSON_VERSION=$(node -e "console.log(require('./package.json').version)")
```

Comment:
I was not able to check how it actually works with tag(because in order to do that I actually need to push a tag), but it at least sets `PUBLISH=Off`